ChemoTracker/components/Profile/ProfileDetailsTable.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import { TabNavigator } from 'react-navigation';
+import Cell from './Cell.js';
+import styles from '../../styles/main.js';
+import color from '../../styles/color.js';
+
+class ProfileDetailsTable extends Component {
+  render() {
+    return (
+      <View style={styles.profileTable}>
+        <Cell title = "Full Name:" description = "Carrie Bradshaw"/>
```

Comment:
I think instead of hardcoding it, you should pass in the `description` as props from `Profile` file (e.g `<ProfileDetailsTable user={userObject}`) where `userObject` is an object that you get from the backend later on and it has the information of the current user
example:
```
userObject: {
    userId:'1234567890',
    fullName: 'Carrie Bradshaw',
    diagnosis: ['Breast Cancer'],
    allergies: [] // this will have the list of allergies, if nothing, then description is 'none',
    ........ //other fields from backend
}
```
then you can use constructor again to catch the props like what you did in `Profile/Cell.js`